editor.planx.uk/src/pages/FlowEditor/components/Team/components/MembersTable.tsx

```diff
@@ -9,14 +11,33 @@ import { AddButton } from "pages/Team";
 import React, { useState } from "react";
 
 import { StyledAvatar, StyledTableRow } from "./../styles";
-import { MembersTableProps } from "./../types";
+import { MembersTableProps, TeamMember } from "./../types";
 import { AddNewEditorModal } from "./AddNewEditorModal";
 
+const TableButton = styled(Button)(({ theme }) => ({
+  color: theme.palette.primary.main,
+  textDecoration: "underline",
+  boxShadow: "none",
+  "&:hover": {
+    boxShadow: "none",
+    color: theme.palette.primary.main,
+    textDecoration: "underline",
+  },
+}));
+
 export const MembersTable = ({
   members,
   showAddMemberButton,
 }: MembersTableProps) => {
-  const [showModal, setShowModal] = useState(false);
+  const [showAddModal, setShowAddModal] = useState(false);
+  const [showUpdateModal, setShowUpdateModal] = useState(false);
```

Comment:
There's two separate States for managing which Modal opens, and it is passed down as a prop to a single modal prop as `setShowModal`